tests/infra_test.go

```diff
@@ -121,19 +126,79 @@ var _ = Describe("Infrastructure", func() {
 			pod := pods.Items[0] // only one compute node in the test environment
 
 			Eventually(func() string {
-				stdout, _, err := tests.ExecuteCommandOnPodV2(virtClient,
-					&pod, "virt-handler",
-					[]string{
-						"curl",
-						"-L",
-						"-k",
-						fmt.Sprintf("https://%s:%s/metrics", pod.Status.PodIP, "8443"),
-					})
-				Expect(err).ToNot(HaveOccurred())
-				lines := filterMetricsOutput(stdout, "kubevirt")
+				lines := getKubevirtVMMetrics(virtClient, &pod, "virt-handler")
 				return strings.Join(lines, "\n")
 			}, 30*time.Second, 2*time.Second).Should(ContainSubstring("kubevirt"))
 		}, 300)
+
+		It("should include the storage metrics for a running VM", func() {
+			By("Creating the VirtualMachineInstance")
+
+			// WARNING: we assume the VM will have a VirtIO disk (vda)
+			vmi := tests.NewRandomVMIWithWatchdog()
+			blockPVName := "block-pv-storage-metrics"
+			tests.CreateBlockVolumePvAndPvc(blockPVName, "1Gi")
+			defer tests.DeletePvAndPvc(blockPVName)
```

Comment:
Could you just add an `emptyDisk`? Then you don't have to do any storage management.

Reply:
Here some background to `emptyDisk`: https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html

Reply:
Neat feature, thanks for the suggestion. Changed accordingly.